### ISSUE TITLE
[UWP] No navigation bar on UWP when using version 3.1.0.637273

### ISSUE THREAD

**Issue by jgold6** [xamarin/Xamarin.Forms#3297]
### Description

On UWP, no navigation bar is shown with Forms version 3.1.0.637273. Works as expected on version 3.1.0.583944.

### Steps to Reproduce

1. Open Visual Studio and create a new Forms solution targeting UWP
2. Update Xamarin Forms package to version 3.1.0.637273
3. Add the following to create the first page:

    ```public partial class App : Application
    {
        public App()
        {
            InitializeComponent();
            var page = new ContentPage { BackgroundColor = Color.Blue, Title = "Page title" };
            MainPage = new NavigationPage(page);
        }
    }

4. Run the UWP app 

### Expected Behavior
Page will show with a navigation bar displaying the page title

### Actual Behavior
Page shows but there is no navigation bar and page title is not displayed

### Basic Information

- Version with issue: 3.1.0.637273
- Last known good version: 3.1.0.583944.
- IDE: Visual Studio 2017
- Platform Target Frameworks: 
    - UWP:  16299 

- Nuget Packages: Forms
- Affected Devices: Windows 10 PC

### Screenshots
Expected result:

![screen shot 2018-07-13 at 10 49 21 am](https://user-images.githubusercontent.com/4606509/42713246-7f322290-868a-11e8-9210-e88a2bd93217.png)

Actual result:

![screen shot 2018-07-13 at 10 51 30 am](https://user-images.githubusercontent.com/4606509/42713307-ba716078-868a-11e8-8307-66137c4b4153.png)

**Comment by CreatingYourApp** [comment]
Any chance to get a fix for this in 3.1? 3.2 seems too far away and still too buggy

**Comment by marciosouzajunior** [comment]
Still not working on 3.1.0.697729

**Comment by samhouts** [comment]
FYI, this was released in the latest service release of 3.1. Thank you!
